Sources/HAP/Server/Device.swift

```diff
@@ -317,6 +317,22 @@ public class Device {
         return pairingState == .paired
     }
 
+    // Remove all the pairings made with this Device
+    // Can be used in the event of a stale configuration file
+    public func unpairFromAllControllers() {
```

Comment:
We don't notify the other party of the removal, so is it really unpairing or removing the pairing?

```suggestion
    public func removeAllPairings() {
```

Reply:
Yes, the HAP protocol doesn't seem to allow for an accessory to request to unpair. The only way to do it is equivalent to a hard reset on a hardware device, i.e to simply close all open connections and forget the pairings. I'll change the function name to your suggestion.